.gitignore

```diff
@@ -107,3 +107,6 @@ node_modules/
 
 # Ignore transpiled JS code
 django_fido/static/django_fido/js
+
+# IDE
```

Comment:
This snuck back in :) But probably a rebase on top of #142 might fix this.

Reply:
That's removed.

Regarding the Backend method signature, 
I am new to the python typing(never seriously used it before) can you please help on this?
```
error: Signature of "authenticate" incompatible with supertype "BaseFido2AuthenticationBackend"
```

I don't want to `@overload` because I don't want overloading but overriding.

Reply:
Your `authenticate` method must take `user` anyway but make it an `Optional[AbstractUser]` which should work.

Reply:
Yes, sorry I forgot the `complete_authentication()` will pass the `user=None` to the backend anyway. All fixed now.

Reply:
Overriding different signature also against LSP anyway so my bad :P